src/Symfony/Component/PropertyInfo/Tests/Fixtures/AdderRemoverDummy.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyInfo\Tests\Fixtures;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class AdderRemoverDummy
+{
+    private $analyses;
+    private $feet;
+
+    public function addAnalyse(Dummy $analyse)
+    {
+    }
+
+    public function removeFoot(Dummy $foot)
```

Comment:
@trousers we've asked you politely to please stop making these comments. You don't listen to us, so we'll need to report this abuse to GitHub: https://github.com/contact?report=trousers